src/otx/algo/hooks/recording_forward_hook.py

```diff
@@ -1,366 +1,395 @@
-# Copyright (C) 2024 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-#
-"""Hooks for recording/updating model internal activations."""
```

Comment:
Please, check that the end of line sequence for this file is `LF`, not `CRLF`
<img width="251" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/33455576/39432bd0-2766-4898-8fe8-7eacea9ab6e4">

Reply:
Thanks, fixed!